res/openvpn-gui-res-it.rc

```diff
@@ -236,7 +236,7 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_ITALIAN, SUBLANG_DEFAULT
 BEGIN
     ICON ID_ICO_APP, ID_ICON_ABOUT, 8, 16, 21, 20
-    LTEXT "OpenVPN GUI v. %ls - Interfaccia OpenVPN per Windows\n\
+    LTEXT "OpenVPN GUI v.%ls - Interfaccia OpenVPN per Windows\n\
```

Comment:
Remove the period (.) after the letter v (see previous comments).
Should be `v%ls ` not `v.%ls`